attributes/default.rb

```diff
@@ -0,0 +1,74 @@
+### Main attributes ###
+default['mapr']['cluster_name'] = 'default-cluster'
+default['mapr']['release']['version'] = '6.0.1'
+default['mapr']['ecosystem']['version'] = '5.0'
+
+default['mapr']['user'] = 'mapr'
+default['mapr']['uid'] = 628
```

Comment:
Why do you need a specific id?

Reply:
It's from MapR documentation : 
    ```The “MapR user” is the user that MapR services run as (typically named mapr or hadoop) on each node. The MapR user has full privileges to administer the cluster. Administrative privilege with varying levels of control can be assigned to other users as well.

Before installing MapR, decide on the name, user id (UID) and group id (GID) for the MapR user. The MapR user must exist on each node, and the user name, UID and primary GID must match on all nodes.```

Reply:
How could guarantee then the uid is not already on all nodes?

Reply:
I've removed this . I will ask to MapR why it's needed and put it back if really needed.